spec/support/service_template_helper.rb

```diff
@@ -131,7 +131,6 @@ def service_template_stubs
       end
 
       def user_helper
-        allow_any_instance_of(User).to receive(:role).and_return("admin")
         @user = FactoryBot.create(:user_with_group, :name => 'Wilma', :userid => 'wilma')
```

Comment:
lets just create an admin user instead of stubbing

```suggestion
        @user = FactoryBot.create(:user_admin)
```

Reply:
@kbrock done.